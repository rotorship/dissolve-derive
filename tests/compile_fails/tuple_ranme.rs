use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct TupleRename(#[dissolved(rename = "name")] String);

fn main() {}
