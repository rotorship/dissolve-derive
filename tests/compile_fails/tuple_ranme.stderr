error: rename is unsupported for tuple struct fields, only skip is allowed
 --> tests/compile_fails/tuple_ranme.rs:4:20
  |
4 | struct TupleRename(#[dissolved(rename = "name")] String);
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
